pkg/sinks/webhook.go

```diff
@@ -20,15 +21,23 @@ type WebhookConfig struct {
 }
 
 func NewWebhook(cfg *WebhookConfig) (Sink, error) {
-	return &Webhook{cfg: cfg}, nil
+	tlsClientConfig, err := setupTLS(&cfg.TLS)
+	if err != nil {
+		return nil, fmt.Errorf("failed to setup TLS: %w", err)
+	}
+	return &Webhook{cfg: cfg, trasport: &http.Transport{
+		Proxy:           http.ProxyFromEnvironment,
+		TLSClientConfig: tlsClientConfig,
+	}}, nil
 }
 
 type Webhook struct {
-	cfg *WebhookConfig
+	cfg      *WebhookConfig
+	trasport *http.Transport
```

Comment:
nit: typo

Reply:
Thanks for the comment. Already fixed in the latest commit.